module-2-html/exercises/01-table-of-contents/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Web design</title>
+        <body>
+            <h1>Web dizajn</h1>
+
+<h2>Uvod u web dizajn</h2>
+<h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h3>
+
+<h2>HTML</h2>
+<h3>Povijest HTML</h3>
+<h4>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h4>
+
+<h3>Sintaksa i pravila pisanja</h3>
+<h4><sin>Sintaksa HTML</sin><br>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h4>
```

Comment:
Što si ovdje htio napisat? Nije mi jasno što predstavlja `<sin>` element, odnosno koji element si htio staviti na njegovo mjesto. Morat ćeš mi to ispraviti.